CHANGELOG.md

```diff
@@ -2,8 +2,26 @@
 
 ## [Unreleased]
 
+### API Breaking
+
+* (Auth) Query for BaseAccount type account is no longer supported and is replaced by EthAccount type. 
+
+### State Machine Breaking
+
+* (IRISHub) [\#2863](https://github.com/irisnet/irishub/pull/2863) Bump up tibc-go version to xxx
+* (IRISHub) [\#2863](https://github.com/irisnet/irishub/pull/2863) Bump up irismod version to xxx
```

Comment:
what is "xxx" ?

Reply:
Tags of these dependencies are not decided yet, so they are temporarily replaced with "xxx".